src/main/java/org/datadog/jmeter/plugins/util/CommonUtils.java

```diff
@@ -0,0 +1,10 @@
+package org.datadog.jmeter.plugins.util;
+
+public class CommonUtils {
+
+    public static String parseThreadGroup(String threadName) {
```

Comment:
```suggestion
    public static String parseThreadGroup(@NonNull  String threadName) {
```
can threadname be null?